code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -42,7 +42,7 @@
 	var/bayNumber //Quick reference to what bay we're in. Usually set to the loading_id variable for the related area type
 	var/customDropoff = FALSE
 	var/picking_dropoff_turf = FALSE
-	var/launchClone = FALSE //If true, then we don't actually launch the thing in the bay. Instead we call duplicateObject() and send the result
+	var/launchClone = FALSE //If true, then we don't actually launch the thing in the bay. Instead we call duplicate_object() and send the result
```

Comment:
eyeballing that camelCase... you fixed it elsewhere. would you be so kind?

Reply:
I didn't want to touch too much outside of just duplication because this seemed like a PR that would be harder to review than normal

Reply:
I figured... I certainly won't block on this, but feel free to if you're feeling helpful

Reply:
There's way too many and it goes into TGUI too, I don't think I want to do it in this PR

![image](https://user-images.githubusercontent.com/53777086/212444970-d8a61d16-c8b2-4018-a1e4-e8caddde1401.png)